src/common/base32z.h

```diff
@@ -21,7 +21,7 @@ const char *encode(std::string const &src, stack_t &stack)
   size_t ret = 0, pos = 1;
   int bits     = 8;
   uint32_t tmp = src[0];
-  size_t len   = sizeof(src);
```

Comment:
Ew, ew, ew.  How did this code even work?

Reply:
:p